src/plat/pc99/linker.lds

```diff
@@ -72,33 +78,40 @@ SECTIONS
 #else
     . = ALIGN(2M);
 #endif
+#else
+    . = ALIGN(8K);
 #endif
 
     ki_boot_end = .;
     ki_skim_start = .;
 
+    . = . + ((ADDR(.boot) + SIZEOF(.boot)) & (8K - 1));
```

Comment:
This and the above ALIGN(8K) requires that the size of .boot is less than 8K?

Reply:
The 8K is because the .skim_bss or .bss section has an alignment of 8k.  This pushes the virt segment up to a 8k alignment. This is where the virt segment starts so we start the virtual addresses at the same offset into an 8K alignment as where the last LOAD section finished.  This results in the .boot and .text sections to be right next to each other in the ELF file but still mapped into different virtual pages while maintaining the required alignments.

Reply:
could you copy/paste resulting addresses of the ELF file to help me understand?